input.js

```diff
@@ -5,13 +5,16 @@ const readline = require('readline')
 const Input = (() => {
 
     const rl = readline.createInterface({
+        completer,
         input: process.stdin,
-        output: process.stdout
+        output: process.stdout,
     });
 
-    rl.on('SIGINT', () => {
-        process.exit()
-    })
+    function completer(line) {
+        const completions = ':help :q :cr :gr'.split(' ')
+        const hits = completions.filter(c => c.indexOf(line) === 0)
```

Comment:
so no leading whitespace?  Not familiar with vim but I suppose it also doesn't allow for leading whitespace?